### ISSUE TITLE
Homebrew formula?

### ISSUE THREAD

**Issue by rlue** [vovkasm/input-source-switcher#6]
Is there any chance we could get a homebrew formula for this project? It'd be nice if it could be installed with 

```
$ brew install vovkasm/utils/issw   # (or similar)
```

**Comment by vovkasm** [comment]
See: https://github.com/Homebrew/legacy-homebrew/pull/38356
May be now it will be appropriate for them ;-)

**Comment by rlue** [comment]
Yes, perhaps! If they don't go for it, it'd still be nice to have it in a custom/personal tap, like the one you have [here](https://github.com/vovkasm/homebrew-gis) (though of course, I hope they do).
